src/main/scala/cromwell/engine/WorkflowDescriptor.scala

```diff
@@ -104,10 +104,10 @@ case class WorkflowDescriptor(id: WorkflowId,
     val logger = backend.workflowLogger(this)
 
     def copyFile(file: WdlFile): Try[Unit] = {
-      val src = file.valueString.toPath(workflowLogger, gcsFilesystem)
-      val wfPath = wfContext.root.toPath(workflowLogger, gcsFilesystem).toAbsolutePath
+      val src = file.valueString.toPath(gcsFilesystem)
+      val wfPath = wfContext.root.toPath(gcsFilesystem).toAbsolutePath
       val relativeFilePath = src.subpath(wfPath.getNameCount, src.getNameCount)
-      val dest = destDirectory.toPath(workflowLogger, gcsOutputsFilesystem).resolve(relativeWorkflowRootPath).resolve(relativeFilePath)
+      val dest = destDirectory.toPath(gcsOutputsFilesystem).resolve(relativeWorkflowRootPath).resolve(relativeFilePath)
```

Comment:
Also pre-existing but I wonder if the double-resolve is necessary or can it somehow be done in one pass